returnsrcvr-checker/src/main/java/org/checkerframework/checker/framework/LombokSupport.java

```diff
@@ -0,0 +1,35 @@
+package org.checkerframework.checker.framework;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ExecutableElement;
+
+import org.checkerframework.checker.returnsrcvr.ReturnsRcvrAnnotatedTypeFactory;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.javacutil.TypesUtils;
+
+/**
+ * 
+ * Lombok support for returns receiver checker
+ *
+ */
+public class LombokSupport implements FrameworkSupport {
+	
+    public boolean knownToReturnThis(AnnotatedTypeMirror.AnnotatedExecutableType t, ReturnsRcvrAnnotatedTypeFactory typeFactory) {
```

Comment:
Here and elsewhere, methods that override a method from a super type should be annotated with `@Override`